### DISCUSSION TITLE
maquetación de páginas

### DISCUSSION THREAD

**Discussion by nataliaramosj** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#188] [General]
Hola compañeros ,  no entiendo a que se refiere con maquetación , no me queda claro si al momento de estar  cumpliendo con la descripción del ejercicio  la maquetación  queda implícitamente hecha o si hay que hacer algo  más , espero puedan ayudarme , gracias .

**Comment by hackit92** [comment]
Con maquetación se refiere a la estructura que estarás realizando en HTML ya que al ser un lenguaje de marcado, únicamente servirá de estructura para el sitio y las tecnologías adicionales que quieras utilizar.

**Reply by nataliaramosj** [reply]
muchas gracias  =)

**Comment by davidtoString** [comment]
Se refiere a crear los archivos html para que ya se vean desde el navegador (las imágenes, lista de productos, formulario).

**Comment by JuanSalvador73** [comment]
Es crear el HTML, tal y como MC lo hizo en un stream online. Para que se vea "bonito" faltan los estilos, pero eso se verá mas adelante. Como sea, estas en libertad de crear el sitio como tu desees solo siguiendo eso si los criterios descritos en la práctica.

Es decir, si tienes conocimientos mas avanzados de WEB puedes hacer el sitio como gustes.
